src/App.css

```diff
@@ -1,3 +1,8 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: aliceblue;
+}
+
```

Comment:
🔧 Fonts are handled in `index.html`. 
I think the background color is handled by the theme.
https://material-ui.com/components/css-baseline/#page